css/evaluations.css

```diff
@@ -0,0 +1,50 @@
+body{
+	background-color: white;
+}
+
+.footer{
+	margin-bottom: 0;
+}
+
+.mainTableClass{
+	width: 60%;
+	margin: 0 auto;
+	margin-top: 100px;
+	margin-bottom: 100px;
+
+	font-size: 20px;
+	border-collapse: collapse;
+}
+
+.mainTableClass th{
+	text-align: left;
+}
+
+.mainTableClass tr{
+	height: 50px;
+
+	border-bottom: 1px solid black;
+}
+
+.mainTableClass th,td{
```

Comment:
What I'm trying to say is that your selector, `.mainTableClass th, td`, targets only `th` elements in an ancestor with a class of `mainTableClass`, but also targets _all_ `td` elements, regardless of whether or not any of their ancestors have a `mainTableClass` class. Case in point:
```html
<table class="mainTableClass">
  <tr>
    <th>Column Header</th>
  </tr>
  <tr>
    <td>Column Cell</td>
  </tr>
</table>
<!-- vs -->
<table> <!-- note I don't have any "mainTableClass" class here -->
  <tr>
    <th>Column Header</th>
  </tr>
  <tr>
    <td>Column Cell</td>
  </tr>
</table>
```